### DISCUSSION TITLE
Problem using gsutil to access metadata

### DISCUSSION THREAD

**Discussion by mwcrepeau** [malariagen/vector-data#54] [Q&A]
I have been unsuccessful in using gsutil to download Ag1000G metadata as described here:

https://malariagen.github.io/vector-data/ag3/download.html#

My commands were:

`mkdir -pv ~/vo_agam_release/v3/metadata/`
`gsutil -m rsync -r gs://vo_agam_release/v3/metadata/ ~/vo_agam_release/v3/metadata/`

and the resulting output was:

`Building synchronization state...`
`Caught non-retryable exception while listing gs://vo_agam_release/v3/metadata/: invalid_grant: Bad Request`
`CommandException: Caught non-retryable exception - aborting rsync`

I tried this first on our HPC server and then on my MacBook, with the same result in both cases.

Any ideas about how I can resolve this problem?

**Comment by ntorres17** [comment]
Hi did you ever figure this out? Was it just a permission issue?

**Comment by mwcrepeau** [comment]
The developers never replied to my GitHub post, so I was never able to resolve the issue.

On Nov 19, 2024, at 12:09 PM, Nancy (Torres) Staeger ***@***.***> wrote:



Hi did you ever figure this out? Was it just a permission issue?

**Reply by ntorres17** [reply]
hi, i figured it out. There is a how-to link and you have to follow that to use it. Hope that helps!

**Comment by mwcrepeau** [comment]
Ok, thanks!  But I’m unclear about where to find the how-to link?

On Dec 10, 2024, at 10:59 AM, Nancy (Torres) Staeger ***@***.***> wrote:



hi, i figured it out. There is a how-to link and you have to follow that to use it. Hope that helps!

**Reply by ntorres17** [reply]
Hi! Im so sorry for the lack of response, holidays. But I used this webiste/link resource

https://malariagen.github.io/vector-data/ag3/cloud.html
